packages/mongodb-rag-core/src/contentStore/EmbeddedContent.ts

```diff
@@ -96,12 +96,9 @@ export interface GetSourcesMatchParams {
   Filters for querying the embedded content vector store.
  */
 export type QueryFilters = {
-  sourceName?: string;
-  version?: {
-    current?: boolean;
-    label?: string;
-  };
-  sourceType?: Page["sourceType"];
+  sourceName?: string | { $in: string[] };
+  version?: { current?: boolean; label?: string | { $in: string[] } };
+  sourceType?: Page["sourceType"] | { $in: string[] };
```

Comment:
since this is a public API, can you make it not coupled to the mongo `$in` operator.

more like:

```suggestion
  sourceName?: string | string;
  version?: { current?: boolean; label?: string | string[] };
  sourceType?: Page["sourceType"] | string[];
```

then have any mapping to mongo be internal to the `MongoDbEmbeddedContentStore`